src/try-convert/Program.cs

```diff
@@ -35,7 +35,7 @@ private static async Task<int> Main(string[] args)
             return await parser.InvokeAsync(args.Length > 0 ? args : new string[] { "-h" }).ConfigureAwait(false);
         }
 
-        public static int Run(string project, string workspace, string msbuildPath, string tfm, bool allowPreviews, bool diffOnly, bool noBackup)
+        public static int Run(string project, string workspace, string? msbuildPath, string tfm, bool allowPreviews, bool diffOnly, bool noBackup)
```

Comment:
so, technically all the strings in this method can be given null and we need to be resilient to it so I would update all of them to be `string?`. I would also update the `AddOption` clauses above to return `new Argument<string?>()` instead of `new Argument<string>()`